### ISSUE TITLE
~~Schema validators --ignoreNiftiHeaders not working~~. n_rows not in for aslcontext association

### ISSUE THREAD

**Issue by rwblair** [bids-standard/legacy-validator#1785]
asl004 and asl003 generate schema.rules.checks.asl.ASLPostLabelingDelayASLContextLength errors even when ignore flag used.

**Comment by effigies** [comment]
Failing check is

```YAML
  checks:
    - length(sidecar.PostLabelingDelay) == associations.aslcontext.n_rows
```

I suspect `associations.aslcontext.n_rows` is currently undefined. At least for ASL003, both values should be 20.

**Comment by effigies** [comment]
Fixed in #1797
